src/main/java/com/example/bookstore/controller/AdminController.java

```diff
@@ -0,0 +1,59 @@
+package com.example.bookstore.controller;
+
+import com.example.bookstore.entity.User;
+import com.example.bookstore.service.UsersService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/admin")
+public class AdminController {
+
+    @Autowired
+    private UsersService usersService;
+
+    @RequestMapping("")
+    public String home() {
+        return "admin/layout";
+    }
+
+    @RequestMapping("/users")
```

Comment:
```suggestion
    @RequestMapping("admin/users")
```
Bất cứ controller nào khai báo trong package admin đều nên xuất phát url là admin nhé thanh niên

Reply:
Da oke chi